app/src/app.js

```diff
@@ -212,6 +224,9 @@ angular.module('kyc-wallet').controller('GuestImportKeystoreController', GuestIm
 const GuestImportPrivateKeyController = requireAppModule('angular/controllers/guest/import/private-key-controller.js');
 angular.module('kyc-wallet').controller('GuestImportPrivateKeyController', GuestImportPrivateKeyController);
 
+const GuestImportLedgerController = requireAppModule('angular/controllers/guest/import/ledger.js');
+angular.module('kyc-wallet').controller('GuestImportLedgerController', GuestImportLedgerController);
+
```

Comment:
this one too, there must be a better way to manage this otherwise the app will be completely unmaintainable after another 6 months of features being added

Reply:
@altninja I don't understand this one, what should i do, every other controllers are implemented like this. You know, the issue is just about ledger integration

Reply:
@tmchedlishvili its ok I'm not requesting changes here just commenting, it won't affect the PR